changelog/fragments/upgrade-kb.yaml

```diff
@@ -0,0 +1,57 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Upgrade controller-runtime (sigs.k8s.io/controller-runtime) dependency version
+      from `v0.6.2` to `v0.6.3`. More info: https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.6.3
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Upgrade controller-runtime version
+      body: >
+        In the `go.mod` file replace `sigs.k8s.io/controller-runtime v0.6.2`
+        with `sigs.k8s.io/controller-runtime v0.6.3` and then run `go mod tidy`.
+  - description: >
+      Upgrade the Kubebuilder CLI dependecy version from `f7a3b65dd250` to `c993a2a221fe`.
+      More info: https://github.com/kubernetes-sigs/kubebuilder/compare/f7a3b65dd250..c993a2a221fe
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Fix the `CRDDirectoryPath` path in the `suite_test.go` files
+      body: >
+        For Golang based-operator projects, fix `CRDDirectoryPath` value in the `controllers/<group>/suite_test.go` files
+        when the project has multi-group support. More info: https://github.com/kubernetes-sigs/kubebuilder/issues/1665
```

Comment:
```suggestion
```

Reply:
HI @estroz, 

Your suggestion here goes against https://github.com/operator-framework/operator-sdk/pull/4062/files#r508071377 and your previous suggestion to add the link for the change.

Reply:
done